src/index.html

```diff
@@ -22,6 +22,405 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a
+        href="/"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__links">
+          <li>
+            <a
+              href="http://"
+              class="nav__link is-active"
+            >
```

Comment:
The href attribute of your anchor tag is not valid. It is recommended to use a valid URL or if you don't have a URL yet, you can use '#' as a placeholder.